src/lib.rs

```diff
@@ -27,324 +27,44 @@ extern crate glutin;
 
 #[macro_use]
 mod macros;
-mod camera;
+pub mod camera;
+pub mod controls;
+pub mod custom;
 mod factory;
+pub mod geometry;
+mod hub;
 mod input;
+pub mod lights;
```

Comment:
`lights::Directional` vs `light::Directional`. I prefer the latter.

Reply:
Should I rename or just reexport with different name?

Reply:
I meant the name of the module - `lights` vs `light`.